src/us/kbase/narrativemethodstore/NarrativeMethodStoreServer.java

```diff
@@ -37,6 +42,12 @@
     public static final String    CFG_PROP_GIT_LOCAL_DIR = "method-spec-git-repo-local-dir";
     public static final String CFG_PROP_GIT_REFRESH_RATE = "method-spec-git-repo-refresh-rate";
     public static final String       CFG_PROP_CACHE_SIZE = "method-spec-cache-size";
+    public static final String         CFG_PROP_TEMP_DIR = "method-spec-temp-dir";
+    public static final String       CFG_PROP_MONGO_HOST = "method-spec-mongo-host";
+    public static final String     CFG_PROP_MONGO_DBNAME = "method-spec-mongo-dbname";
+    public static final String       CFG_PROP_MONGO_USER = "method-spec-mongo-user";
+    public static final String   CFG_PROP_MONGO_PASSWORD = "method-spec-mongo-password";
+    public static final String      CFG_PROP_ADMIN_USERS = "method-spec-admin-users";
     
     public static final String VERSION = "0.2.6";
```

Comment:
Let's update to 0.3.0-alpha, so we can easily check that this version was deployed.  Can you also add a new section to the release notes as well with a release date TBD.  Thanks.